userspace/otpilot/src/spi_processor.rs

```diff
@@ -43,6 +43,18 @@ use spiutils::protocol::wire::FromWireError;
 use spiutils::protocol::wire::ToWire;
 use spiutils::protocol::wire::ToWireError;
 
+// Size of the SPI flash chip.
+// Hard-coded to 64 MiB for now.
+// TODO: Modify this to be read from the actual SPI flash chip at runtime.
```

Comment:
Yeah you're right. We actually usually create bugs/issues, since they're trackable. I'll need to sort that out on this repo. For now, I just added my username as per Google convention.